packages/react/__tests__/__snapshots__/exports.ts.snap

```diff
@@ -69,6 +68,8 @@ Array [
   "ToggleButtonGroup",
   "View",
   "VisuallyHidden",
+  "classNameModifier",
+  "classNameModifierByFlag",
```

Comment:
Can we avoid adding these as public exports?

Reply:
We could make copies of them or move them to ui package

Reply:
I can move them to `/internal` ? I mean _reallllly_ these should live in `@aws-amplify/ui` so we can share them with other web frameworks